app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user && user.authenticate(params[:session][:password])
```

Comment:
`user && user.authenticate(params[:session][:password])` thành `user&.authenticate(params[:session][:password])`

dọc thêm http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/